src/workerd/io/worker.c++

```diff
@@ -1494,7 +1494,7 @@ kj::Maybe<jsg::JsObject> tryResolveMainModule(jsg::Lock& js,
     // call to tryResolveModuleNamespace because I intend to add some additional
     // logging/metrics logic around this call.
     KJ_IF_SOME(ns,
-        jsg::modules::ModuleRegistry::tryResolveModuleNamespace(js, mainModule.toString(true))) {
```

Comment:
Why?

Reply:
`mainModule.toString(true)` means the generated string will have a leading `/`, which we don't actually want here. This was a bug found while I was working in here.